docs/src/test/scala/docs/http/scaladsl/server/HttpsServerExampleSpec.scala

```diff
@@ -61,12 +61,12 @@ abstract class HttpsServerExampleSpec extends AnyWordSpec with Matchers
     //#both-https-and-http
     // you can run both HTTP and HTTPS in the same application as follows:
     val commonRoutes: Route = get { complete("Hello world!") }
-    Http().newServerAt("127.0.0.1", 443).enableHttps(https).bindFlow(commonRoutes)
-    Http().newServerAt("127.0.0.1", 80).bindFlow(commonRoutes)
+    Http().newServerAt("127.0.0.1", 443).enableHttps(https).bind(commonRoutes)
+    Http().newServerAt("127.0.0.1", 80).bind(commonRoutes)
```

Comment:
:+1: weird that we had `bindFlow` here before

Reply:
I think that might have been missed by scalafix for some reason.